jit-persistence/src/main/java/org/dcsa/jit/persistence/entity/TransportCall.java

```diff
@@ -20,10 +28,46 @@
 @AllArgsConstructor
 @Setter(AccessLevel.PRIVATE)
 @Entity
-@Table(name = "")
+@Table(name = "transport_call")
 public class TransportCall {
   @Id
   @GeneratedValue
   @Column(name = "id", nullable = false)
   private UUID id;
+
+  @Column(name = "transport_call_reference", length = 100, nullable = false)
+  private String reference; // Really a UUID
```

Comment:
The reference field is not guaranteed to the be UUID in the standard as it is right now.  The fact that the reference implementation always uses an UUID is an implementation detail rather than a standard specification.